packages/cosmo-dycore/package.py

```diff
@@ -45,6 +45,7 @@ class CosmoDycore(CMakePackage):
     version('dev-build', branch='master')
     version('mch', git='git@github.com:MeteoSwiss-APN/cosmo.git', branch='mch')
     version('c2sm', git='git@github.com:C2SM-RCM/cosmo.git', branch='master')
+    version('c2sm-features', git='git@github.com:C2SM-RCM/cosmo.git', branch='master_additional_features')
```

Comment:
I would have called the branch master_c2sm_features or c2sm-features for consistency, but if you already use the name in other scripts I guess it is difficult to change.

Reply:
Good idea. @mjaehn and I thought long about the name. What do you think Michael?